# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1323664321 18000
#      Sun Dec 11 23:32:01 2011 -0500
# Node ID a8952515415d03392d00be30600f6f1f23b615f0
# Parent  2ed17790425e98430574eac3319990338384f598
# Parent  08569c50df6f08d8297d466d3eba76d613de2fa4
maint: periodic merge of stable to default

diff --git a/src/base-list.h b/liboctave/base-list.h
rename from src/base-list.h
rename to liboctave/base-list.h
